src/sentry/api/endpoints/issues/related_issues.py

```diff
@@ -10,6 +11,12 @@
 from sentry.types.ratelimit import RateLimit, RateLimitCategory
 
 
+class RequestSerializer(serializers.Serializer):  # type: ignore[type-arg]
```

Comment:
idk why my comment didn't post here in the first revision I'll try and rewrite it again!

Serializers come in two forms, one where they are registered for a particular type and another where they are just used to accept view parameters -- this is the latter one and so the generic type can just be `[None]`